packages/core/react/src/WalletProvider.tsx

```diff
@@ -6,6 +6,7 @@ import {
     SolanaMobileWalletAdapterWalletName,
 } from '@solana-mobile/wallet-adapter-mobile';
 import { type Adapter, type WalletError, type WalletName } from '@solana/wallet-adapter-base';
+import type { WalletAdapter } from '@solana/wallet-adapter-base/src/adapter.js';
```

Comment:
```suggestion
import { type WalletAdapter } from '@solana/wallet-adapter-base';
```

However this import should be combined with the existing import above it. Make sure not to import from files in an external package.